examples/apps/quick_tap.py

```diff
@@ -0,0 +1,463 @@
+        await self.cozmo_ready_position()
+        self.print_starting_instructions()
+        if not self.cubes_connected():
+            print('Cubes did not connect successfully - check that they are nearby. You may need to replace the batteries.')
+            return
+        await self.assign_cubes()
+        self.quick_tap_state = GAME_STATE
+        while max(self.player.score, self.cozmo_player.score) < SCORE_TO_WIN:
+            await self.game_round()
+        await self.report_winner()
+
+    async def game_round(self):
+        '''Sets up and runs a round of the game. 
+
+        In run(), a new round starts unless a player's score reaches SCORE_TO_WIN.
+
+        First we ready the players and cubes, and then start the countdown.
+        After the countdown, the cubes light up.  Then Cozmo makes his move.
+        Once Cozmo's move is over, we determine the winner of the round,
+        and Cozmo reacts accordingly.
+        '''
+        self.round_over = False
+        await self.reset_players()
+        await self.countdown_cube.countdown()
+        await self.set_round_lights()
+        self.round_start_time = time.time()
+        await self.cozmo_player.determine_move(self.buzzer_display_type)
+        while not self.round_over: # self.round_over is True when Cozmo's tap animation is completed
+            await asyncio.sleep(0)
+        await self.cozmo_anim_reaction()
+
+    async def set_round_lights(self):
+        '''Waits a random delay, then sets a display on the buzzer cubes.'''
+        await self.cube_light_delay()
+        self.determine_buzzer_display()
+        self.set_buzzer_lights()
+
+    async def reset_players(self):
+        '''Gets the players and cubes ready for a new round.'''
+        self.player.reset()
+        self.cozmo_player.reset()
+        await self.robot.set_lift_height(1.0).wait_for_completed()
+        self.turn_off_buzzer_cubes()
+        
+    async def cube_light_delay(self):
+        '''Waits between 0 and 2 seconds.'''
+        delay = random.random() * 2
+        await asyncio.sleep(delay)
+
+    def determine_buzzer_display(self):
+        '''Chooses a buzzer display type based on the probabilities defined above.'''
+        probability_red = random.random()
+        if probability_red < RATE_MAKE_BUZZERS_RED:
+            self.buzzer_display_type = MAKE_BUZZERS_RED
+        else:
+            probability_different_colors = random.random()
+            if probability_different_colors < RATE_MAKE_BUZZERS_DIFFERENT_COLORS:
+                self.buzzer_display_type = MAKE_BUZZERS_DIFFERENT_COLORS
+            else:
+                self.buzzer_display_type = MAKE_BUZZERS_SAME_COLORS
+
+    def on_cube_tap(self, evt, obj, **kwargs):
+        '''Responds to cube taps depending on quick_tap_state.
+
+        If in CHOOSE_CUBES_STATE, on_cube_tap assigns the player's cube.
+        If in GAME_STATE, on_cube_tap registers the tap time of the players.
+        '''
+        if obj.object_id:
```

Comment:
should probably be "is not None", otherwise this would also be false for a zero ID ( which I assume isn't what you're testing for).